.github/workflows/commitlint.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: Commitlint
+on:
+  push:
+    branches-ignore:
+      - "main"
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    env:
+      CI: "true"
```

Comment:
This is set by default :)

https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/

Reply:
Thanks. Removed.